masonry/src/event.rs

```diff
@@ -50,6 +50,84 @@ pub enum PointerButton {
     Other,
 }
 
+/// A set of [`PointerButton`]s.
+#[derive(PartialEq, Eq, Clone, Copy, Default)]
+pub struct PointerButtons(u8);
```

Comment:
Is it worth implementing `From<PointerButton> for PointerButtons`?

Reply:
Unsure again since we barely actually. use this.